app-frontend/src/app/pages/library/buckets/buckets.html

```diff
@@ -0,0 +1,110 @@
+<div class="row content stack-sm">
+  <div class="column">
+    <h1 class="h3">Buckets</h1>
+    <form action="">
+      <div class="form-group all-in-one">
+        <label for="name"><i class="icon-bucket"></i></label>
+        <input id="name" type="text" class="form-control" placeholder="New bucket name">
+        <button class="btn btn-link">Create</button>
+      </div>
+    </form>
+    <div class="list-group">
+      <div class="list-group-item">
+        <a class="list-group-link" href="/bucket-view-list.html"></a>
```

Comment:
There are several href links which won't work, and should be changed to srefs if the page is available.